packages/core/bootstrap/src/lib/middleware/cache/index.ts

```diff
@@ -18,18 +18,26 @@ import * as redis from './redis'
 import * as metrics from './metrics'
 import { CacheOptions, CacheImplOptions, CacheEntry, Cache } from './types'
 
-const DEFAULT_CACHE_KEY_GROUP = uuid()
+const UUID = uuid()
 
 export const defaultOptions = (
   shouldUseLocal?: boolean,
   adapterContext?: AdapterContext,
 ): CacheOptions => {
+  let cacheGroupKey = ''
+  if (adapterContext?.name) {
+    cacheGroupKey =
+      adapterContext?.name + (getEnv('CACHE_KEY_GROUP') ? `-${getEnv('CACHE_KEY_GROUP')}` : '')
+  } else {
+    cacheGroupKey = UUID + (getEnv('CACHE_KEY_GROUP') ? `-${getEnv('CACHE_KEY_GROUP')}` : '')
+  }
```

Comment:
Could we simplify this?

```suggestion
  const cacheGroupKey = `${adapterContext?.name || UUID}-${ getEnv('CACHE_KEY_GROUP') || '' }`
```

I don't think it would be a problem that there's an extra `-` at the end in most cases.